spec/fixtures/vcr_cassettes/Auth0_Api_V2_Anomaly/_check_if_ip_is_blocked/should_rerurn_200_response_code.yml

```diff
@@ -0,0 +1,65 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://auth0-sdk-tests.auth0.com/api/v2/anomaly/blocks/ips/192.0.2.0
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept:
+      - "*/*"
+      Accept-Encoding:
+      - gzip, deflate
+      User-Agent:
+      - rest-client/2.0.2 (darwin18.6.0 x86_64) ruby/2.6.3p62
+      Content-Type:
+      - application/json
+      Auth0-Client:
+      - eyJuYW1lIjoicnVieS1hdXRoMCIsInZlcnNpb24iOiI0LjUuMCJ9
```

Comment:
Your attention to detail is commendable!

I'll make a note to anonymize this in our existing cassettes. Nothing secret here but probably best not to have specific application data in here.

Reply:
Thank you for your advice!  
However, this value has been updated to the same value as the test already included in the master.
Ex:
[should_add_a_token_to_the_blacklist.yml:19](https://github.com/auth0/ruby-auth0/blob/master/spec/fixtures/vcr_cassettes/Auth0_Api_V2_Blacklists/_add_token_to_blacklist/should_add_a_token_to_the_blacklist.yml#L19)

You might want to include client id in filter_sensitive_data as well.

spec/spec_helper.rb
``` 
config.filter_sensitive_data('CLIENT_SECRET') { ENV['CLIENT_SECRET'] }
config.filter_sensitive_data('API_TOKEN') { ENV['MASTER_JWT'] }
```

Reply:
I think that's probably the best way to deal with it. My main reason for commenting was that I appreciate you updating to the same value as the rest 👍